backend/app/Policies/UserPolicy.php

```diff
@@ -0,0 +1,105 @@
+<?php
+
+namespace App\Policies;
+
+use App\Models\Permission;
+use App\Models\User;
+use Illuminate\Auth\Access\HandlesAuthorization;
+
+class UserPolicy
+{
+    use HandlesAuthorization;
+
+    /**
+     * Determine whether the user can view any models.
+     *
+     * @param  \App\Models\User  $user
+     * @return mixed
+     */
+    public function viewAny(User $user)
+    {
+        //
+    }
+
+    /**
+     * Determine whether the user can view the model.
+     *
+     * @param  \App\Models\User  $user
+     * @param  \App\Models\User  $model
+     * @return mixed
+     */
+    public function view(User $user, User $model)
+    {
+        //
+    }
+
+    /**
+     * Determine whether the user can create models.
+     *
+     * @param  \App\Models\User  $user
+     * @return mixed
+     */
+    public function create(User $user)
+    {
+        //
+    }
+
+    /**
+     * Determine whether the user can update the model.
+     *
+     * @param  \App\Models\User  $user
+     * @param  \App\Models\User  $model
+     * @return Boolean
+     */
+    public function update(User $user, User $model)
+    {
+        $is_updating_own_user = $user->id === $model->id;
+
+        if (!$is_updating_own_user
+            && $user->can(Permission::UPDATE_USERS)
+        ) {
+            return true;
+        }
+
+        // TODO: Check if user can update user within own publication
+
+        return $is_updating_own_user;
+
```

Comment:
```suggestion
        //User is updating their own record.
        if ($user->id === $model->id) {
            return true;
        }
        //User has global permission to update users
        if ($user->can(Permission::UPDATE_USERS)) {
            return true;
        }

        // TODO: Check if user can update user within own publication
        // No explicit permission so return false.
        return false;
```
I feel like this is a little easier to digest.